packages/wdio-testingbot-service/tests/__mocks__/testingbot-tunnel-launcher.ts

```diff
@@ -1,4 +1,4 @@
-export default (opts: any, cb: any) => {
+export = (opts: any, cb: any) => {
     cb(null, {
```

Comment:
@christian-bromann This was what made me confused. The mock for Jest uses `export default`, so I was confused.

Reply:
So if it's OK, I'd propose to fix the unit test mocking for consistency in this PR. I'll revert unrelated codes.

Reply:
Why does the mock need to be changed? `export default` is correct here as it mimics the way how testingbot exports its service: `module.exports = require('./build/tunnel-launcher')`